src/index.ts

```diff
@@ -86,6 +96,32 @@ const setupServer = async () => {
 
         const result = await addUser(userData);
         return result;
+      },
+      async login(_, args) {
+        console.log(args.data);
+        const user = await AppDataSource.manager.findOneBy(User, {
+          email: args.data.email
+        });
+        console.log(user);
+        if (!user) {
+          throw new CustomError('Unable to login', 404);
+        }
+        const isMatch = await bcrypt.compare(args.data.password, user.password);
+        if (!isMatch) {
+          throw new CustomError('Unable to login', 404);
+        }
+        return {
+          user: {
+            id: user.id,
+            name: user.name,
+            email: user.email,
+            birthDate: user.birthDate
+          },
```

Comment:
```suggestion
        return {
          user,
```

Dá pra usar esse atalho porque mesmo que o objeto `user` tenha mais campos, o GraphQL já cuida de filtrar apenas os campos presentes no schema declarado.